src/io/flutter/sdk/FlutterSettingsConfigurable.form

```diff
@@ -76,9 +76,9 @@
               <toolTipText value="Report anonymized usage information to Google Analytics."/>
             </properties>
           </component>
-          <component id="99e58" class="javax.swing.JLabel" binding="myPrivacyPolicy">
+          <component id="c59f0" class="com.intellij.ui.components.labels.LinkLabel" binding="myPrivacyPolicy">
             <constraints>
-              <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="3" hsize-policy="3" anchor="0" fill="1" indent="3" use-parent-layout="false"/>
+              <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="3" hsize-policy="3" anchor="8" fill="0" indent="3" use-parent-layout="false"/>
```

Comment:
Just checking that the <grid> changes are intentional?

Reply:
Not completely - their GUI editor is super fiddly w/ creating changes. I'll revert some of them locally and see if the layout still looks good.

Reply:
Reverted most of the changes; the layout still looks good.